tests/acceptance/features/steps/keystone_token.py

```diff
@@ -111,36 +172,74 @@ def step_get_list_trial_users(context):
     """
     try:
-        result = context.expiredusers.gerlisttrialusers()
+        result = context.expiredusers.get_trial_users()
         print('\n      Number of trial users found: {}\n\n'.format(len(result)))
     except ValueError:
         assert False, 'Cannot recover the list of trial users'
 
 
-@when(u'I request a list of expired users')
+@when(u'I request a list of expired trial users')
 def step_request_list_expired_users(context):
     """
     From the list of trial users, returns the list of expired users.
     :param context: Context of the acceptance test execution.
     :return: Nothing.
     """
-    context.expiredusers.finalList = context.expiredusers.get_list_expired_users()
+    result = context.expiredusers.get_list_expired_trial_users()
+    print ("in the result")
+    print(result)
+    context.expiredusers.finalTrialList = result
+    print (context.expiredusers.finalTrialList)
 
 
-@then(u'the component returns a list with all the expired trial users')
-def step_get_list_expired_users(context):
+@then(u'the component returns a list with "{number}" expired trial users')
+def step_get_list_expired_users(context, number):
     """
     Recover the list of expired trial users.
     :param context: Context of the acceptance test execution.
     :return: Nothing.
     """
     try:
-        result = context.expiredusers.finalList
+        result = context.expiredusers.finalTrialList
+
         print('\n      Number of expired trial users found: {}\n\n'.format(len(result)))
+        assert result is not None, \
+            'Expected a valid list expired users'
+        assert len(result) == int(number), \
+            'Expected a list of users with {0} length and found {1}'.format(number, len(result))
+
     except ValueError:
         assert False, 'Cannot recover the list of trial users'
 
 
+@when(u'I request a list of expired community users')
+def step_request_list_expired_users(context):
+    """
+    From the list of trial users, returns the list of expired users.
+    :param context: Context of the acceptance test execution.
+    :return: Nothing.
+    """
+    context.expiredusers.finalCommunityList = context.expiredusers.get_list_expired_community_users()
+
+
+@then(u'the component returns a list with "{number}" expired community users')
+def step_get_list_expired_users(context, number):
+    """
+    Recover the list of expired trial users.
+    :param context: Context of the acceptance test execution.
+    :return: Nothing.
+    """
+    try:
+        result = context.expiredusers.finalCommunityList
+        print('\n      Number of expired community users found: {}\n\n'.format(len(result)))
+        assert result is not None, 'Expected a valid list expired users'
+        assert len(result) == int(number), \
+            'Expected a list of users with {0} length and found {1}'.format(number, len(result))
+
+    except ValueError:
+        assert False, 'Cannot recover the list of community users'
+
+
```

Comment:
I think that some code duplication is in this file, you should check it. Last but not least, I miss some method description.